content-security-policy/connect-src/connect-src-json-import-blocked.sub.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>connect-src-json-import-blocked</title>
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="connect-src 'none'; script-src 'self' 'unsafe-inline';"
+    />
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+  </head>
+
+  <body>
+    <script>
+      promise_test((t) => {
+        let spv = new Promise((resolve, reject) => {
+          window.addEventListener("securitypolicyviolation", (e) => {
+            if (e.blockedURI.endsWith("dummy.json")) {
+              resolve();
+            } else {
+              reject();
```

Comment:
I would not reject here, but rather just ignore, because there might be unrelated violations fired (see my other comment).